doc/releases/migration-guide-4.3.rst

```diff
@@ -64,9 +64,10 @@ Ethernet
 Networking
 **********
 
-.. zephyr-keep-sorted-start re(^\w)
-
-.. zephyr-keep-sorted-stop
```

Comment:
The sorted options were just recently added and the other subsystems have those too so I think we should not remove these.

Reply:
@pdgendgt

Reply:
@cfriedt @pdgendt 
How should I fix it?
Looking at recent release notes https://github.com/zephyrproject-rtos/zephyr/blob/main/doc/releases/migration-guide-4.2.rst under Networking there are directly bullet points.
Does the sorting skip it and just uses the letters after that?

So should I do the following? Or what's the expected structure?
```
.. zephyr-keep-sorted-start re(^\w)

* The HTTP server now respects the configured ``_config`` value. Check that
  you provide applicable value to :c:macro:`HTTP_SERVICE_DEFINE_EMPTY`,
  :c:macro:`HTTPS_SERVICE_DEFINE_EMPTY`, :c:macro:`HTTP_SERVICE_DEFINE` and
  :c:macro:`HTTPS_SERVICE_DEFINE`.

.. zephyr-keep-sorted-stop
```

Reply:
@pdgendt @cfriedt 
Done